bundle/edu.gemini.wdba.xmlrpc.server/src/main/java/edu/gemini/wdba/tcc/TccNames.java

```diff
@@ -23,6 +23,7 @@ private TccNames() {
     public static final String AZEL = "AzEl";
     public static final String BASE = "Base";
 //    public static final String BASE = "sciencetarget";
+    public static final String BAGSTARGET = "bagsTarget";
```

Comment:
@jluhrs needs to review this. I believe we don't need to expose the `bagsTargets` to the TCC.